go/libkb/sig_chain.go

```diff
@@ -665,6 +665,10 @@ func (l *SigChainLoader) LoadLinksFromStorage() (err error) {
 		if link, err = ImportLinkFromStorage(curr, suid, l.G()); err != nil {
 			return
 		}
+		if link == nil {
```

Comment:
I think the error gets returned on line 665 above. This just handles the additional case where the link is nil despite no error being returned.